ssa/testdata/test6.yaml

```diff
@@ -4,7 +4,7 @@ kind: Namespace
 metadata:
   name: "%[1]s"
 ---
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2
```

Comment:
Do we want/need to continue to test `autoscaling/v2beta2`? Seems like this is potentially introducing a blind spot.